internal/command/scale/count.go

```diff
@@ -38,6 +38,25 @@ For pricing, see https://fly.io/docs/about/pricing/`
 		flag.String{Name: "process-group", Description: "The process group to scale"},
 		flag.Bool{Name: "with-new-volumes", Description: "New machines each get a new volumes even if there are unattached volumes available"},
 		flag.String{Name: "from-snapshot", Description: "New volumes are restored from snapshot, use 'last' for most recent snapshot. The default is an empty volume"},
+		flag.String{
+			Name:        "vm-size",
+			Description: `The VM size to use when deploying for the first time. See "fly platform vm-sizes" for valid values`,
+			Aliases:     []string{"size"},
+		},
+		flag.Int{
+			Name:        "vm-cpus",
+			Description: "Number of CPUs",
+			Aliases:     []string{"cpus"},
+		},
+		flag.String{
+			Name:        "vm-cpukind",
+			Description: "The kind of CPU to use ('shared' or 'performance')",
+		},
+		flag.Int{
+			Name:        "vm-memory",
+			Description: "Memory (in megabytes) to attribute to the VM",
+			Aliases:     []string{"memory"},
+		},
```

Comment:
this set of flags keeps repeating, would be good to merge them into a `Flag.Set{}` and reuse it from all commands that require VM size related flags.